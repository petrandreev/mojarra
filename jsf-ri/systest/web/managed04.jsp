<!--
 Copyright 2004 Sun Microsystems, Inc. All rights reserved.
 SUN PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
-->

<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
  <head>
    <title>Verify property ordering</title>
    <%@ taglib uri="http://java.sun.com/jsf/core"  prefix="f" %>
    <%@ taglib uri="http://java.sun.com/jsf/html"  prefix="h" %>
  </head>

  <body>
    <h1>Verify property ordering</h1>

<f:view>

  <h:form>
 
    <h:outputText value="#{propertyOrder.order}" /> 
    <h:outputText value="#{propertyOrder.listProperty[0]}" />
    <h:outputText value="#{propertyOrder.listProperty[1]}" />
    <h:outputText value="#{propertyOrder.listProperty[2]}" />
    <h:outputText value="#{propertyOrder.listProperty[3]}" />
   
  </h:form>

</f:view>

    <hr>
  </body>
</html>
