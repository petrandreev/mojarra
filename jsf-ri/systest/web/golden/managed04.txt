<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
  <head>
    <title>Verify property ordering</title>
    
    
  </head>

  <body>
    <h1>Verify property ordering</h1>


<form id="j_id_id15" name="j_id_id15" method="post" action="/jsf-systest/faces/managed04.jsp;jsessionid=b76d0071bbe2d0ec398b04d50039" enctype="application/x-www-form-urlencoded">
<input type="hidden" name="j_id_id15" value="j_id_id15" />
<input type="hidden" name="javax.faces.ViewState" id="javax.faces.ViewState" value="j_id85:j_id86" />
 one two three fourfivesixseveneight
</form>

    <hr>
  </body>
</html>
